cmd/resource-quota-check/resource-quota.yaml

```diff
@@ -6,19 +6,17 @@ metadata:
   namespace: kuberhealthy
 spec:
   runInterval: 30m
-  timeout: &resource_quota_check_timeout 5m
+  timeout: 5m
   podSpec:
     containers:
     - name: resource-quota
-      image: kuberhealthy/resource-quota-check:v1.1.1
+      image: kuberhealthy/resource-quota-check:v1.2.2
```

Comment:
can this just be 1.2.0?

Reply:
https://hub.docker.com/repository/registry-1.docker.io/kuberhealthy/resource-quota-check/tags?page=1 1.2.1 was built and pushed a while ago